org.freedesktop.tuhi.json

```diff
@@ -0,0 +1,106 @@
+{
+    "id": "org.freedesktop.tuhi",
+    "command": "/app/bin/tuhi",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "cleanup": [ "/cache",
+                 "/include",
+                 "/lib/pkgconfig",
+                 "/man",
+                 "/share/aclocal",
+                 "/share/devhelp",
+                 "/share/gir-1.0",
+                 "/share/gtk-doc",
+                 "/share/man",
+                 "/share/pkgconfig",
+                 "/share/vala",
+                 "/lib/systemd",
+                 "*.la", "*.a" ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
+    "finish-args": [
+        "--system-talk-name=org.bluez",
+        "--own-name=org.freedesktop.tuhi1"
+    ],
+    "modules": [
+        {
+            "name": "cpython",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.python.org/ftp/python/3.6.4/Python-3.6.4.tar.xz",
+                    "sha256": "159b932bf56aeaa76fd66e7420522d8c8853d486b8567c459b84fe2ed13bcaba"
+                }
+            ]
+        },
+        {
+            "name": "pycairo",
```

Comment:
I'm assuming this is pulled in by one of the other dependencies?

Reply:
Unfortunately yes, by pygobject.

If we want to package tuhi-kete too, we will at least also need to package svgwriter in the flatpak BTW